### DISCUSSION TITLE
后台录制的时候时不时就会自动取消录制，到最后录下来好几个断断续续的文件

### DISCUSSION THREAD

**Discussion by yimoulaiye** [HMBSbige/BilibiliLiveRecordDownLoader#317] [Q&A] [answered]
![{C6E6E7F5-04BD-4b59-A23D-B12DD6E9C2E5}](https://github.com/HMBSbige/BilibiliLiveRecordDownLoader/assets/154862531/c8eabe45-696f-4955-a144-b96fc41459c9)

**Comment by HMBSbige** [comment, marked as answer]
看日志，应该是自身网络不稳定亦或是B站限制导致的

可以尝试登录或者手动调整直播间高级设置（如编码格式）
